distribution/src/assemble/component.xml

```diff
@@ -18,42 +18,127 @@
     limitations under the License.
 
 -->
-<assembly
-  xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
-  <id>binary-release</id>
-  <formats>
-    <format>tar.gz</format>
-    <format>dir</format>
-  </formats>
-  <includeBaseDirectory>true</includeBaseDirectory>
+<component xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
   <moduleSets>
+    <moduleSet>
+      <useAllReactorProjects>true</useAllReactorProjects>
+     <!--Now, select which projects to include in this module-set. -->
+    <includes>
+      <include>org.apache.drill.exec:drill-jdbc:jar</include>
+      <include>org.apache.drill:drill-protocol:jar</include>
+      <include>org.apache.drill:drill-common:jar</include>
+      <include>org.apache.drill:drill-logical:jar</include>
+      <include>org.apache.drill.exec:vector:jar</include>
+      <include>org.apache.drill.memory:drill-memory-base:jar</include>
+      <include>org.apache.drill.exec:drill-rpc:jar</include>
+      <include>org.apache.drill.exec:drill-java-exec:jar</include>
+      <include>org.apache.drill.contrib.storage-hive:drill-storage-hive-core:jar</include>
+      <include>org.apache.drill.contrib.storage-hive:drill-hive-exec-shaded:jar</include>
+      <include>org.apache.drill.contrib.data:tpch-sample-data:jar</include>
+      <include>org.apache.drill.contrib:drill-mongo-storage:jar</include>
+      <include>org.apache.drill.contrib:drill-storage-hbase:jar</include>
+      <include>org.apache.drill.contrib:drill-format-mapr:jar</include>
+      <include>org.apache.drill.contrib:drill-jdbc-storage:jar</include>
+      <include>org.apache.drill.contrib:drill-kudu-storage:jar</include>
+      <include>org.apache.drill.contrib:drill-storage-kafka:jar</include>
+      <include>org.apache.drill.contrib:drill-opentsdb-storage:jar</include>
+      <include>org.apache.drill.contrib:drill-udfs:jar</include>
+    </includes>
+      <sources>
+        <includeModuleDirectory>false</includeModuleDirectory>
+        <fileSets>
+          <fileSet>
+            <directory>target</directory>
+            <outputDirectory>/jars</outputDirectory>
```

Comment:
should this be `jars` only instead of `/jars` as specified in `outputDirectory` of `dependencySet` ?